src/main/java/ch/uzh/ifi/hase/soprafs22/entity/User.java

```diff
@@ -118,4 +137,9 @@ public String getGenderString(){
     }
 
     public void setGender(Gender gender){this.gender = gender; }
+
+    public BlackCard getBlackCard(){return this.blackCard; }
+
+    public void setBlackCard(BlackCard blackCard){this.blackCard = blackCard; }
+
```

Comment:
don't we also need getters and setters for the white cards?

Reply:
yes we doI i will add this as soon we need them